scripts/fuzz_opt.py

```diff
@@ -947,30 +947,39 @@ def can_compare_to_others(self):
 
     def handle_pair(self, input, before_wasm, after_wasm, opts):
         global ignored_vm_runs
-        ignored_before = ignored_vm_runs
 
         before = self.run_vms(before_wasm)
 
-        # if the binaryen interpreter hit a host limitation on the original
-        # testcase, or for some other reason we need to ignore this, then stop
-        # (otherwise, a host limitation on say allocations may be hit in the
-        # 'before' but not in the 'after' as optimizations may remove it).
-        if before[self.bynterpreter] == IGNORE:
-            # the ignoring should have been noted during run_vms()
-            assert(ignored_vm_runs > ignored_before)
-            return
```

Comment:
Do we still need this logic as well in case the reason for the host limitation is optimized out?

Reply:
This function calls `run_vms` twice, which this PR fixes. It also calls `compare_before_and_after`, which are passed the results we do have (which will be only the binaryen interpreter, so that should be ok).